File: VocaDbModel/Helpers/AlbumHelper.cs
using VocaDb.Model.Domain;
using VocaDb.Model.Domain.Albums;

namespace VocaDb.Model.Helpers
{
	public static class AlbumHelper
	{
		public static ContentFocus GetContentFocus(DiscType t) => t switch
		{
			DiscType.Artbook => ContentFocus.Illustration,
			DiscType.Video => ContentFocus.Video,
			_ => ContentFocus.Music,
		};
	}
}

File: VocaDbWeb/New/Helpers/AlbumHelper.ts
import { AlbumType } from '@/types/Models/Albums/AlbumType';
import { ContentFocus } from '@/types/Models/ContentFocus';

export class AlbumHelper {
	static getContentFocus = (albumType: AlbumType): ContentFocus => {
		switch (albumType) {
			case AlbumType.Artbook:
				return ContentFocus.Illustration;

			case AlbumType.Video:
				return ContentFocus.Video;

			default:
				return ContentFocus.Music;
		}
	};
}
